db/seeds.rb

```diff
@@ -165,50 +159,159 @@
 User.create!(
   email: 'tupac@gmail.com',
   password: 'password',
-  nickname: 'QO',
+  nickname: 'Grubby',
   show_game: csgo,
   game: csgo,
   rank: Rank.find_by(id: 42),
   position_name_id: csgo.position_names.first.id)
 
 puts 'Users created'
 
-
 puts 'Creating Teams'
 
-Team.create!(
-  name: 'Team Secret',
+team_logo = URI.open('https://res.cloudinary.com/dm4nbcc5v/image/upload/v1584528718/logos_for_seeds/chaos_pgwdak.png')
```

Comment:
how are we accessing cloudinary images without an API key?

Reply:
Basically it is a two step process:
1. I uploaded manually the logos to cloudinary (stored in the separate logos_for_seeds folder);
2. I am seeding them in from there (and they get attached and re-uploaded to cloudinary's main folder);

This seems the only way to do it that worked. There is no way to just attach the manually uploaded ones for some unknown reason.

Reply:
Haha.... Wow, that's incredibly annoying!